src/content/en/updates/2019/06/chromium-chronicle-3

```diff
@@ -0,0 +1,64 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The Chromium Chronicle, a monthly series geared specifically to Chromium developers — the developers who build the browser.
+
+{# wf_updated_on: 2019-06-18 #}
+{# wf_published_on: 2019-06-25 #}
+{# wf_tags: chromium-chronicle #}
+{# wf_featured_image: /web/updates/images/generic/cr-chron.jpg #}
+{# wf_featured_snippet: Tests are critical because they find bugs and regressions, enforce better designs and make code easier to maintain. This month, we take a look at how to conduct thorough tests with Gerrit #}
+{# wf_blink_components: N/A #}
+
+<style>
```

Comment:
You can remove this style block since we're not showing any code in this article

Reply:
Done!